app/src/main/kotlin/com/simplemobiletools/smsmessenger/extensions/Context.kt

```diff
@@ -1023,6 +1023,20 @@ fun Context.subscriptionManagerCompat(): SubscriptionManager {
     }
 }
 
+fun Context.renameConversation(conversation: Conversation, newTitle: String): Conversation {
+    require(conversation.isGroupConversation) {
+        "Can only rename group conversations."
```

Comment:
whats this, it will crash with the message?

Reply:
it won't crash now, it'll crash when somebody in the future changes the code to rename non-group conversation.

Reply:
ah, we dont need such checks..